Controller/DefaultController.php

```diff
@@ -40,7 +39,7 @@ public function indexAction(Request $request)
         }
 
         $results = null;
-        if ($term != '') {
+        if ('' != $term) {
```

Comment:
wat is de reden dat die van plaats zijn gewisseld ?
vind $term != '' gelijk logischer als je het leest